cmake/templates/NSIS.template.in

```diff
@@ -161,7 +161,7 @@
 ;General
 
   ; hide install details since we show an image slideshow in their place
-  ShowInstDetails nevershow
+  ShowInstDetails hide
```

Comment:
What does this do?

Reply:
This is the part of the installer where the slideshow used to appear. Now, it shows the "Show Details" button common to NSIS installers.